taipy/core/_manager/_manager.py

```diff
@@ -154,7 +154,7 @@ def _delete_entities_of_multiple_types(cls, _entity_ids: _EntityIds):
         _SubmissionManagerFactory._build_manager()._delete_many(_entity_ids.submission_ids)
 
     @classmethod
-    def _export(cls, id: str, folder_path: Union[str, pathlib.Path]):
+    def _export(cls, id: str, folder_path: Union[str, pathlib.Path], **kwargs):
         """
         Export an entity.
         """
```

Comment:
It should be up to date if we want to keep this docstring (Which is not mandatory since the function is not exposed as a public API). Otherwise, we can remove it.